src/Options.js

```diff
@@ -156,6 +157,7 @@ addFeatureOption('generatorComprehension', EXPERIMENTAL);
 addFeatureOption('memberVariables', EXPERIMENTAL);
 addFeatureOption('require', EXPERIMENTAL);
 addFeatureOption('strongMode', EXPERIMENTAL);
+addFeatureOption('strongModeRuntime', EXPERIMENTAL);
```

Comment:
With `strongModeRuntime` I was expecting a runtime function addition.  I guess you are trying express "some runtime costs".  `strongModeChecks` or `strongModeArityChecks`?